server/apps/embed.js

```diff
@@ -42,6 +42,30 @@ const queryReports = async () => {
                         name
                     }
                 }
+                testPlan(id: $testPlanDirectory) {
+                    testPlanVersions {
+                        id
+                        title
+                        phase
+                        testPlanReports(isFinal: true) {
+                            id
+                            metrics
+                            at {
+                                id
+                                name
+                            }
+                            browser {
+                                id
+                                name
+                            }
+                            latestAtVersionReleasedAt {
+                                id
+                                name
+                                releasedAt
+                            }
+                        }
+                    }
+                }
                 testPlanReports(
```

Comment:
Is it possible to remove this part of the query?

Reply:
I don't think so. `testPlanReports` is being used in the code.

Reply:
As I said in another comment, you're right that it's being used. My point is more that it's only being used by dead code and the whole network of dead code present in this file can be removed.